data_pipeline/in_memory_queue.py

```
@@ -74,17 +76,17 @@ def is_json_serializable(o):
     try:
         json.dumps(o)
         return True
-    except:
-        pass
+    except Exception as e:
+        print(e)
     return False
 
```

will the function return false if an exception is raised? or will the function just terminate on the exception.

---

The function should only return true if the try succeeds and won't need to print exceptions so that is why the return True is within the try. If it fails, the goal is to continue running and just return false. By printing out the exception it resolves bare except ruff errors.

---

I understand what you are saying. There isn't anything wrong with your logic, however it's "Best Practice" to not have a return in the try clause.

---

Ah ok I see what you mean